torchquad/integration/utils.py

```diff
@@ -206,3 +207,19 @@ def _torch_repeat(a, repeats, axis=None):
     # torch.repeat_interleave corresponds to np.repeat and should not be
     # confused with torch.Tensor.repeat.
     return torch.repeat_interleave(a, repeats, dim=axis)
+
+@partial(register_function, "torch", 'expand_dims')
+def _torch_expand_dims(a, axis):
```

Comment:
could you add a comment / docstring what this function does and why it was added? :)

Reply:
done!